packages/eslint-config-motley/package.json

```diff
@@ -24,7 +24,7 @@
     "eslint": "^7.7.0",
     "eslint-config-airbnb": "18.2.0",
     "eslint-config-airbnb-base": "14.2.0",
-    "eslint-config-prettier": "^6.11.0",
+    "eslint-config-prettier": "^8.1.0",
```

Comment:
https://github.com/motleyagency/eslint-config-motley/blob/master/packages/eslint-config-motley-typescript/.eslintrc.js#L5

Does this need to be removed too? Probably as it's already in the main config.

Reply:
Right, good catch.